ci-operator/step-registry/cucushift/installer/check/vsphere/disk/cucushift-installer-check-vsphere-disk-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source "${SHARED_DIR}/govc.sh"
+source "${SHARED_DIR}/vsphere_context.sh"
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+declare vsphere_datacenter
+declare vsphere_datastore
+INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+check_result=0
+
+function check_disk_type() {
+    local ret=0
+    mapfile -t  vm_lists < <(govc ls /${vsphere_datacenter}/vm/${INFRA_ID})
```

Comment:
It is better to ensure the `vm_lists` is not empty, or else, throw error.

Reply:
ok